pkg/commands/compute/serve.go

```diff
@@ -810,43 +812,76 @@ func (c *pushpinContext) buildPushpinConf() string {
 // command line and/or fastly.toml. The cleanup function on the returned pushpinContext
 // needs to eventually be called by the caller to shut down Pushpin.
 func (c *ServeCommand) startPushpin(spinner text.Spinner, out io.Writer) (pushpinContext, error) {
+	text.Output(out, "%s: %s", text.BoldYellow("EXPERIMENTAL"), "Enabling Pushpin support for local testing of Fanout.")
```

Comment:
Use `text.Important` instead of duplicating its functionality.

Reply:
text.Important puts the heading IMPORTANT but I wanted the heading EXPERIMENTAL.

For now I'll switch it to `Info`